en/guide/configuration.md

```diff
@@ -101,3 +101,22 @@ This option lets you define the custom directories of your Nuxt.js application.
 This option lets you define the default properties of the page transitions.
 
 [Documentation about `transition` integration](/api/configuration-transition)
+
+
+## Asynchronous Configuration
+
+If you need to populate some options (e.g.&nbsp;the head) with asynchronous data (e.g.&nbsp;coming from an API), you have the possibility to return a promise. Here is an example:
```

Comment:
```suggestion
If you need to populate some options (e.g. the head) with asynchronous data (e.g. coming from an API), you have the possibility to return a promise. Here is an example:
```

Reply:
Could you also mention that axios has to be configured "again"/"manually"! and can't be used via the module because we are still in the config?